src/sentry/static/sentry/app/views/groupTags.jsx

```diff
@@ -8,32 +9,51 @@ import LoadingError from '../components/loadingError';
 import LoadingIndicator from '../components/loadingIndicator';
 import {percent, deviceNameMapper} from '../utils';
 import {t} from '../locale';
+import withEnvironment from '../utils/withEnvironment';
 
 const GroupTags = createReactClass({
   displayName: 'GroupTags',
+
+  propTypes: {
+    environment: PropTypes.object,
```

Comment:
maybe define this shape in https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/proptypes.jsx?

Reply:
👍 Will open a separate PR and fix it everywhere

Reply:
https://github.com/getsentry/sentry/pull/7558

Reply:
awesome, thanks